### ISSUE TITLE
From Canonical Correlation Analysis to Self-supervised Graph Neural Networks

### ISSUE THREAD

**Issue by SerialLain3170** [SerialLain3170/nagato#76]
## Overview
- Authors propose a self-supervised learning model for graph data with the standpoint of Canonical Correlation Analysis (CCA)
  - CCA-based self-supervised learning has loss function that consists of invariance term and decorrelation term
  - The two terms lead to minimizing multi-view information bottleneck
  - The method has no reliance on negative samples, projector and asymmetric architecture

![Screen Shot 2021-11-02 at 20 07 55](https://user-images.githubusercontent.com/32360147/139835657-26efd5b0-925c-4c23-882a-b73567dbae2f.png)

**Comment by SerialLain3170** [comment]
## Evaluation
- The method is competitive to not only other unsupervised methods but also supervised methods

![Screen Shot 2021-11-02 at 20 15 01](https://user-images.githubusercontent.com/32360147/139836549-116bb5d3-40e7-4261-9ca3-ce537e45a4dc.png)

**Comment by SerialLain3170** [comment]
## Supplemental Information

[Paper Link](https://arxiv.org/pdf/2106.12484.pdf)

### Authors
Hengrui Zhang, Qitian Wu, Junchi Yan, David Wipf, Philip S. Yu

### Organization
Department of Computer Science, University of Illinois at Chicago
Department of Computer Science and Engineering, Shanghai Jiao Tong University
AWS Shanghai AI Lab

### Conference
NeurIPS 2021
